### PR TITLE
Split Omnibus F4 pro and non-pro into two boards 

### PR DESCRIPTION
**Pull request by Hwurzburg**
Allows Pro version to have 8 PWM outputs for apps like Quadplane. Merely changed name of omnibusf4pro to omnibusf4, added back same file with changes to disable inverter control on UART6 and use those pads as PWM7 and 8, as omnibusf4pro.
Also have PR for WIKI doc updates
Thanks to Basti (vierfuffzig) for implementing and testing the PWM7/8 code change.

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat [hunk 1]
```diff
@@ -70,8 +72,8 @@ PC12 SPI3_MOSI SPI3
 PC11 SPI3_MISO SPI3
 PC10 SPI3_SCK SPI3
 
-PB5 LED OUTPUT HIGH GPIO(57)
-PB4 TIM3_CH1 TIM3 GPIO(56) ALARM
+PB5 LED OUTPUT HIGH GPIO(59)
```
Comment (sh83) [inline libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat]: Please change this too
define HAL_GPIO_A_LED_PIN 57

File: libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat [hunk 2]
```diff
@@ -81,10 +83,10 @@ PC5 VBUS INPUT OPENDRAIN
 # Overwise UART1 or UART6 or RCIN will not work
 PB8 TIM4_CH3 TIM4 RCININT PULLDOWN LOW
 
-#Omnibus F4 V3 and later had hw inverter on UART6
+#Omnibus F4 AIO V3 and later had hw inverter on UART6
 #Overide it to use as GPS UART port
-PC8 SBUS_INVERT_RX OUTPUT LOW
-PC9 SBUS_INVERT_TX OUTPUT LOW
+#PC8 SBUS_INVERT_RX OUTPUT LOW
```
Comment (sh83) [inline libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat]: If we split into different targets, it is better to remove this, not to comment.

File: libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat]: Will this conflict with PC9 definition above which is GPIO(57)? should a similar change be made to the omnibusF4 hwdef.dat

Comment (sh83) [reply libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat]: It is not conflict. LED driver will just use gpio 57, which is now pwm pin. So led will not work. It should be define HAL_GPIO_A_LED_PIN 59 to match pin definition.
Omnibusf4 do not need this change, cause you did not change numbering.

**Comment by vierfuffzig** [comment]
maybe use include instead of adding full hwdefs with just two pins reassigned?

File: libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat (hunk 2 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat]: will make those changes....am using the "include" method for the OmnibusF4 board....but before I push the changes, I want to try to see if I can make the board LED actually show the arming status, currently, it flashes during the bootloader , then is inactive from then on...

**Comment by Hwurzburg** [comment]
Made changes suggested by reviewers and enhanced board LED to not only indicate bootloader activity, but also function as BLUE led (primarily arming status indications)

File: libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat [hunk 3]
```diff
@@ -70,8 +72,12 @@ PC12 SPI3_MOSI SPI3
...
 PC10 SPI3_SCK SPI3
 
-PB5 LED OUTPUT HIGH GPIO(57)
-PB4 TIM3_CH1 TIM3 GPIO(56) ALARM
+
+PB5 LED_BLUE OUTPUT LOW GPIO(0)
+define HAL_GPIO_A_LED_PIN 0
+define HAL_GPIO_B_LED_PIN 1 #dummy assignment required to allow AP_NOTIFY to use board led
+
+#PB4 TIM3_CH1 TIM3 GPIO(58) ALARM
```
Comment (sh83) [inline libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat]: Why this commented out?

File: libraries/AP_HAL_ChibiOS/hwdef/omnibusf4/hwdef.dat [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+include ../omnibusf4pro.dat
+
+undef PC8
+undef PC9
+
+#Omnibus F4 AIO V3 and later had hw inverter on UART6
+#Overide it to use as GPS UART port
+PC8 SBUS_INVERT_RX OUTPUT LOW
+PC9 SBUS_INVERT_TX OUTPUT LOW
+
```
Comment (sh83) [inline libraries/AP_HAL_ChibiOS/hwdef/omnibusf4/hwdef.dat]: BOARD_PWM_COUNT_DEFAULT probably also should be redefined

File: libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat (hunk 3 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/omnibusf4pro/hwdef.dat]: mistaken remnant of attempts to get led to work...corrected...

File: libraries/AP_HAL_ChibiOS/hwdef/omnibusf4/hwdef.dat (hunk 4 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/omnibusf4/hwdef.dat]: will do...also I found a conflict with RSSI which I corrected...

**Comment by Hwurzburg** [comment]
btw...flight tested https://www.youtube.com/watch?v=qgGsuVUFDBE

**Comment by tridge** [comment]
darn, merged before checking if it needs squashing and re-wording msgs

**Pull request merged**